packages/calling-stateful-client/src/CallAgentDeclarativeCommon.ts

```diff
@@ -37,7 +44,7 @@ export abstract class ProxyCallAgentCommon {
   private _internalContext: InternalCallContext;
   private _callSubscribers: Map<CallCommon, CallSubscriber>;
   private _incomingCallSubscribers: Map<string, IncomingCallSubscriber>;
-  private _declarativeIncomingCalls: Map<string, DeclarativeIncomingCall>;
+  private _declarativeIncomingCalls: Map<string, DeclarativeIncomingCall | DeclarativeTeamsIncomingCall>;
```

Comment:
Is it possible to make this incominghCallCommon? There should be a code sample to convert common to a specific type, you can search for isTeamsCall function (lemme know if I remember a wrong name)

Reply:
https://github.com/Azure/communication-ui-library/blob/ac957d3ca983ce34e1d8dd2e3f14cb5aff409c45/packages/calling-stateful-client/src/TypeGuards.ts#L28

Found it

Reply:
will check!